pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -290,6 +301,8 @@ func (pc *PodController) handleRemovePod(key string) error {
 		} else {
 			// Delete cache entry from podCNIInfo.
 			pc.podCache.DeleteCNIConfigInfo(containerInfo)
+			// Delete Pod specific VF cache (if one exists)
+			pc.deleteVFDeviceIDListPerPod(containerInfo.PodName, containerInfo.PodNameSpace)
```

Comment:
Question - could we pass `key` argument of handleRemovePod() to deleteVFDeviceIDListPerPod()

Reply:
Yes, key from the function argument can be used as well.